weis/frequency/raft_wrapper.py

```diff
@@ -28,6 +28,7 @@ def setup(self):
         frequencies = np.arange(min_freq, max_freq+0.5*min_freq, min_freq)
         raft_opt['nfreq'] = len(frequencies)
         raft_opt['n_cases'] = weis_opt['DLC_driver']['n_cases']
+        raft_opt['intersection_mesh'] = 0
```

Comment:
@yqliaohk - This is required by the new RAFT v1.4, but I'm not sure if there should be some more logic involved than hard coding to 0 (or 1).

Reply:
That is helpful to see, thank you!  Looks like the connect to this `raft_wrapper.py` file hasn't been completed yet, so let's merge this one first, and then you can edit this hard-coded 0-value with the new modeling options variable you created (after you merge in the recent changes to `develop` once this PR is complete).